mesonbuild/backend/backends.py

```diff
@@ -81,13 +82,13 @@ def __init__(self, name, fname, cmd_args, env, is_cross, exe_wrapper,
         self.capture = capture
 
 class TestSerialisation:
-    def __init__(self, name, project, suite, fname, is_cross_built, exe_wrapper, is_parallel,
+    def __init__(self, name, project, suite, fname, for_machine: MachineChoice, exe_wrapper, is_parallel,
```

Comment:
I believe since the syntax is from an earlier proposal, they won't do anything but do indeed parse on 3.5. I kept them as documentation, then. I cam still remove them if you like.